.eslintrc.js

```diff
@@ -19,6 +22,14 @@ module.exports = {
     '@typescript-eslint',
     'unicorn',
   ],
+  overrides: [
+    {
+      files: 'src/react/**/__examples__/*',
+      rules: {
+        'react/display-name': 'off',
+      },
+    },
+  ],
```

Comment:
Open to discussion... It's being triggered in the example because the function is unnamed. We could also rewrite the example as either:

```tsx
import { BackLink } from '../BackLink'

export default function Example() {
  return <BackLink>Back</BackLink>
}
```

or:

```tsx
import { BackLink } from '../BackLink'

export const Example = () => <BackLink>Back</BackLink>
```

and have `examples-site/react/example-client.tsx` look for that export.

There are probably other options too...

It might be better to satisfy the rule so that we don't have code snippets that fail lint rules.

Reply:
I like the name export version